### DISCUSSION TITLE
How to get YOLO badge? 2022

### DISCUSSION THREAD

**Discussion by WilliamWelll** [community/community#30197] [Other Feature Feedback, Questions, & Ideas]
How to get YOLO badge? 2022

**Comment by jfullstackdev** [comment]
this is my very own badge, it states

![image](https://user-images.githubusercontent.com/47092464/185749887-2efddb54-78eb-4659-a694-d7f67e003717.png)

**Comment by BergerAPI** [comment]
Looking at [The Achievement-List](https://github.com/Schweinepriester/github-profile-achievements), you will need to merge a pull request without doing a code-review beforehand. So basically, you just need to merge a pull request.

**Comment by NikhilMath** [comment]
Here is a list of how to get all the badges
[Badges List](https://github.com/Schweinepriester/github-profile-achievements)

> Merged a pull request without code review

So have a friend do a pull request and approve it without coding a review.

**Comment by MahaSegni** [comment]
I asked the same question, I had to get someone else to review the pull request

**Comment by Rayane-T** [comment]
Check [Achievement List](https://github.com/Schweinepriester/github-profile-achievements) !

**Comment by Itsfoss0** [comment]
You just  have to merge a  PR without code review. 
Follow [This](https://github.com/community/community/discussions/18611#discussioncomment-3165946)

**Comment by hariprasd** [comment]
> How to get YOLO badge? 2022

I've just got it, Try to do a PR & merge it w/o any comments/reviews

**Comment by SaptarshiSarkar12** [comment]
Just merge a Pull request without any review

**Comment by Kushal997-das** [comment]
Hello I got the answer and it's perfectly working -

STEP 1 : Choose a normal repo. of yours remember in that repo there shouldn't be any `.github` folder or any auto running things not present .

STEP 2 : Now go to that repository settings add another account as collaborator . (Settings->collaborator->Add people).

STEP 3 : You have to Create a Pull request (PR) on this repository.

STEP 4 : Add that collaborator account as reviewer.

STEP 5 : Then You have to merge your opened PR from Your account only not collaborator , Without reviewing the PR .

I hope it gonna help you guys , Thanks :)

**Comment by nsafarova** [comment]
1. Create a new pull request in your repo and open it
2. In the top right corner at Reviewers section, request someone to review your pull request
3. Merge the pull request without finishing the review.

**Comment by YasinDehfuli** [comment]
This Course will help you to get Yolo Badge Step by Step : 

https://github.com/4xmen/Get-Github-Achievements/blob/main/assets/steps/README.yolo.md

If you need more help to how to get all github badges, use this link and learn step-by-step and by image : 

https://github.com/4xmen/Get-Github-Achievements

Hope it helpful to you and you get Yolo badge sooner ;
